packages/ui/src/theme/overwrites/components/textarea.ts

```diff
@@ -0,0 +1,31 @@
+export const Textarea = {
+  baseStyle: {
+    control: {
+      _disabled: {
```

Comment:
Idk why when I tried to apply _focus : {boxShadow: 'outline'} it didn't work and I had to put it in the Textarea component itself
Maybe someone else has a clue what I did wrong

Reply:
I had a problem with that when I was creating the Input component. Probably some styles couldn't be set as base, but only as styles for the selected variant. You can try to set styles for outline, e.g .:
```suggestion
export const Textarea = {
  baseStyle: {
    width: '12.5rem',
    height: '5rem',
  },
  sizes: {
    sm: {
      px: '0.75rem',
      py: '0.375rem',
    },
    md: {
      px: '1rem',
      py: '0.5rem',
    },
    lg: {
      px: '1rem',
      py: '0.625rem',
    },
    defaultProps: {
      size: 'md',
    },
  },
  variants: {
    outline: {
      _disabled: {
        opacity: '0.4',
      },
      _focus: {
        boxShadow: 'outline',
      },
      _hover: {
        borderColor: 'gray.400',
      },
    },
  },
};

```

Reply:
Great, thanks!